src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] hashArray = new Node[DEFAULT_INITIAL_CAPACITY];
+    private Node<K, V> temporaryNode;
```

Comment:
don't put internal variables as class fields

Reply:
Both variables are used in several methods